actionpack/lib/action_controller/metal/redirecting.rb

```diff
@@ -124,6 +129,20 @@ def _extract_redirect_to_status(options, response_options)
         end
       end
 
+      def _extract_redirect_to_response_body(options, response_options)
+        if options.is_a?(Hash) && options.key?(:response_body)
+          options.delete(:response_body)
```

Comment:
breaking change:

`response_body` in `options` argument was used at `url_for(options)` in `_extract_redirect_to_status` before this change.
This line deletes it from `options`, so we may need deprecation warning before the change is applied.